scripts/skills/perks/perk_rf_feral_rage.nut

```diff
@@ -38,25 +38,25 @@ this.perk_rf_feral_rage <- ::inherit("scripts/skills/skill", {
 				id = 10,
 				type = "text",
 				icon = "ui/icons/regular_damage.png",
-				text = "[color=" + ::Const.UI.Color.PositiveValue + "]+" + this.m.RageStacks + "[/color] Damage in Melee"
+				text = "[color=" + ::Const.UI.Color.PositiveValue + "]+" + this.m.RageStacks + "%[/color] increased melee damage"
 			},
 			{
 				id = 11,
 				type = "text",
 				icon = "ui/icons/melee_defense.png",
-				text = "Only receive [color=" + ::Const.UI.Color.PositiveValue + "]" + (100 - ::Math.min(70, 2 * this.m.RageStacks)) + "%[/color] of incoming damage"
+				text = "Only receive [color=" + ::Const.UI.Color.PositiveValue + "]" + (100 - ::Math.min(70, 3 * this.m.RageStacks)) + "%[/color] of incoming damage"
```

Comment:
The damage-received calculation should be its own utility function as it is used in two places.
```
function getDamageReceivedMult()
{
	return ::Math.maxf(0.3, 1.0 - this.m.RageStacks * 0.03)
}
```
Ideally by also introducing new member variables for `DamageReductionPerStack` and `MinimumDamageReceivedMult`